src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -145,10 +165,11 @@ private static void OnExit(object sender, EventArgs e)
 
         try
         {
-            _tracerProvider.Dispose();
+            _tracerProvider?.Dispose();
+            _meterProvider?.Dispose();
             _sdkEventListener.Dispose();
 
-            Logger.Information("OpenTelemetry tracer exit.");
+            Logger.Information("OpenTelemetry tracer/meter exit.");
```

Comment:
When we include logs, it will be OpenTelemetry tracer/meter/logger exit.

Maybe just OpenTelemetry Automatic Instrumentation exit ?